qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -609,6 +609,9 @@ def from_backend(
 
         Args:
             backend: The ``Backend`` instance to build the :class:`.DynamicsBackend` from.
+               Note that even if the `Backend` is :class:`~qiskit.providers.backend.BackendV2`,
+               only :class:`~qiskit.providers.backend.BackendV2` with ``configuration`` and
+               ``defaults`` can be used.
```

Comment:
```suggestion
               Note that while the type hint indicates that `backend` should be a :class:`~qiskit.providers.backend.BackendV1` instance,
               this method also works for :class:`~qiskit.providers.backend.BackendV2` instances that have been set up with sufficiently populated ``configuration`` and ``defaults`` for backwards compatibility.
```

Just a slight suggestion on wording. This may mess up the linting.

Reply:
Thank you for your correction.